src/extras-openmoji/climate-environment/E2C1.svg

```diff
@@ -0,0 +1,21 @@
+<svg id="emoji" viewBox="0 0 72 72" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Also too big, maybe do the same circle for the background

Reply:
Not better with circle IMO ... will leave as it is. Sorry!

![2020-02-27 at 10 27](https://user-images.githubusercontent.com/480224/75430611-db9adc80-594b-11ea-9ae8-7ee14bde40da.jpg)